### PR TITLE
Ug general commands

### LINKED ISSUE #253: update all screenshots to the latest ui

### PR DESCRIPTION
**Pull request by iamjamestan**
Fixes AY2021S1-CS2103T-T15-1/tp#253

### CODE REVIEW

File: src/main/resources/view/AppointmentListCard.fxml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 <?import javafx.scene.layout.VBox?>
 
 <?import javafx.scene.layout.Region?>
-<BorderPane id="borderPane" fx:id="borderPane" maxHeight="175" maxWidth="Infinity" minHeight="100" minWidth="300" prefHeight="160" prefWidth="400" xmlns="http://javafx.com/javafx/8.0.172-ea" xmlns:fx="http://javafx.com/fxml/1" styleClass="appt_card">
+<BorderPane id="borderPane" fx:id="borderPane" maxHeight="175" maxWidth="Infinity" minHeight="100" minWidth="300" prefHeight="160" prefWidth="300" xmlns="http://javafx.com/javafx/8.0.172-ea" xmlns:fx="http://javafx.com/fxml/1" styleClass="appt_card">
     <padding>
         <Insets bottom="5" left="20" right="5" top="5" />
```
Comment (ya0-yuan) [inline src/main/resources/view/AppointmentListCard.fxml:14]: i realised that the string will still overflow when it hits the maximum number of characters and when the scroll bar appears, can you help change the left and right margin to 5 instead?

File: src/main/resources/view/ClientListCard.fxml [hunk 2]
```diff
@@ -10,7 +10,7 @@
 <?import javafx.scene.layout.VBox?>
```
Comment (ya0-yuan) [inline src/main/resources/view/ClientListCard.fxml:11]: same for this class line 24

File: src/main/resources/view/HairdresserListCard.fxml [hunk 3]
```diff
@@ -10,7 +10,7 @@
 <?import javafx.scene.layout.VBox?>
 
 <?import javafx.scene.layout.Region?>
-<BorderPane VBox.vgrow="ALWAYS" id="borderPane" fx:id="borderPane" maxWidth="Infinity" maxHeight="800" minWidth="400" prefWidth="400" xmlns="http://javafx.com/javafx/8.0.172-ea" xmlns:fx="http://javafx.com/fxml/1" styleClass="hairdresser_card">
+<BorderPane VBox.vgrow="ALWAYS" id="borderPane" fx:id="borderPane" maxWidth="Infinity" maxHeight="800" minWidth="300" prefWidth="300" xmlns="http://javafx.com/javafx/8.0.172-ea" xmlns:fx="http://javafx.com/fxml/1" styleClass="hairdresser_card">
     <minHeight>
         <Region fx:constant="USE_COMPUTED_SIZE"></Region>
     </minHeight>
```
Comment (ya0-yuan) [inline src/main/resources/view/HairdresserListCard.fxml:16]: same here in line 24

File: src/main/resources/view/AppointmentListCard.fxml (hunk 1 above)
Comment (ya0-yuan) [inline src/main/resources/view/AppointmentListCard.fxml]: i realised that the string will still overflow when it hits the maximum number of characters and when the scroll bar appears, can you help change the left and right padding to 5 instead?

Comment (iamjamestan) [reply src/main/resources/view/AppointmentListCard.fxml]: Do you mean padding instead of margin?

**Comment by ya0-yuan** [comment]
LGTM！

File: src/main/resources/view/AppointmentListCard.fxml (hunk 1 above)
Comment (ya0-yuan) [reply src/main/resources/view/AppointmentListCard.fxml]: yes! sorry for the typo

**Pull request merged**